google/cloud/ndb/model.py

```diff
@@ -1978,6 +1979,50 @@ def _deserialize(self, entity, p, unused_depth=1):
         """
         raise exceptions.NoLongerImplementedError()
 
+    def _legacy_deserialize(self, entity, p, unused_depth=1):
+        # Ported from legacy NDB, used for decoding pickle properties.
+        """Internal helper to deserialize this property from a protocol buffer.
+
+        Subclasses may override this method.
+
+        Args:
```

Comment:
FWIW, I try to stick to this (more or less) for docstrings. 
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

In this case, the args should be indented and given types.